src/tcgwars/logic/impl/pokemod/PokemodFossil.groovy

```diff
@@ -130,129 +182,871 @@ public enum PokemodFossil implements LogicCardInfo {
   public Card getImplementation() {
     switch (this) {
       case AERODACTYL_1:
-        break
+        return evolution (this, from:"Mysterious Fossil", hp:HP060, type:FIGHTING, retreatCost:2) {
+          weakness GRASS
+          resistance FIGHTING, MINUS30
+          pokemonPower "Prehistoric Power", {
+            text "No more Evolution cards can be played. This power stops working while Aerodactyl is Asleep, Confused, or Paralyzed."
+            delayedA {
+              if (self.cards.filterByType(POKEMON_TOOL)) {
+              before PLAY_EVOLUTION, {
+                if(!self.specialConditions){
+                  wcu "Prehistoric Power prevents you from playing Evolution cards!"
+                  prevent()
+                }
+              }
+            }
+            }
+          }
+          move "Wing Attack", {
+            text "30 damage."
+            energyCost C, C, C
+            attackRequirement {}
+            onAttack {
+              damage 30
+            }
+          }
```

Comment:
This is a pokeBody in Pokémod, not a Pokémon Power. You should also update the "text" value where appropriate to match what is on the Pokémod card. Keeping the text accurate helps with reviewing the changes to ensure they are accurate to the text on the card without having to load the carddb in the browser.